### ISSUE TITLE
Target being treated as passive

### ISSUE THREAD

**Issue by GiovanniL19** [ember-mobile/ember-cli-fastclick#8]
Hi,

I have the following code: 
`<div class="row" {{action "do" something}}></div>`

The first time I click the action it works, anytime after it does nothing and prints the warning message to the console. The warning reads:

> Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080

The line which is trggering the error is on line 446 of fastclick.js 
`event.preventDefault();`

Does anyone have any ideas?

**Comment by lawitschka** [comment]
I don't know about this problem in particular, but I remember problems with
FastClick and (supposedly) non-clickable elements.

Can you try giving the div `cursor: pointer`? Just a wild guess, but worth
a try :)

**Comment by GiovanniL19** [comment]
Hi @lawitschka , 
Thanks for your reply.  Sadly giving the div "cursor:pointer" doesn't work :/

I also made the div a button and that makes no difference. Strange thing is, when I remove fastclick from the project, the action does nothing. 

From testing, this issue it is not a fastclick issue. It was caused by the action being inside an ember component. I solved the issue by removing the action from inside the component and placing it outside.
